packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorBlue600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  medium: 6,
+  large: 12
+};
+
+const sizeToBorderRadius = size => heights[size] / 2;
+
+const ProgressBackground = styled.div.attrs(props => ({
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  margin: ${zdSpacingXs} 0;
+  border-radius: ${({ borderRadius }) => borderRadius}px;
+  background-color: ${zdColorGrey200};
+  color: ${({ color }) => color};
+`;
+
+const ProgressIndicator = styled.div.attrs(props => ({
+  height: heights[props.size],
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  transition: width 0.1s ease-in-out;
+  border-radius: ${({ borderRadius }) => borderRadius}px;
+  background: currentColor;
+  width: ${({ value }) => value}%;
+  height: ${({ height }) => height}px;
+`;
+
+const Progress = ({ value, size, ...other }) => {
+  const percentage = Math.max(0, Math.min(100, value));
+
+  return (
+    <ProgressBackground
+      data-garden-id={COMPONENT_ID}
+      data-garden-version={PACKAGE_VERSION}
+      aria-valuemax="100"
+      aria-valuemin="0"
+      aria-valuenow={percentage}
+      role="progressbar"
+      size={size}
+      {...other}
+    >
+      <ProgressIndicator value={percentage} size={size} />
+    </ProgressBackground>
+  );
+};
+
+Progress.propTypes = {
+  color: PropTypes.string.isRequired,
+  /** The progress as a value between 0 and 100 */
+  value: PropTypes.number.isRequired,
+  size: PropTypes.oneOf(['small', 'medium', 'large'])
+};
+
+Progress.defaultProps = {
+  value: 0,
+  color: zdColorBlue600,
```

Comment:
The default `color` seems bit strange on the prop sheet because it includes the `require` statement.

<img width="965" alt="Screen Shot 2019-10-02 at 10 54 35 AM" src="https://user-images.githubusercontent.com/1811365/66069716-ca68b380-e504-11e9-97a3-c0e449aae5f0.png">

Also, do you think the default color should be `inherit`? Similar to the default color of the `<Spinner>` component?

Reply:
I talked with @jzempel about it, we agreed to use blue as the default color. Should I maybe inline the color code, or default it inside of the code instead?

Reply:
Awesome sounds good! I think inlining the color code or defaulting it in the code would be nice if it makes the prop sheet more reader friendly (e.g. without the `require`) statement.